app/services/application_service.rb

```diff
@@ -0,0 +1,5 @@
+class ApplicationService
```

Comment:
let's discuss

Reply:
Sir, removed this module, so use as of now.